java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -1400,7 +1400,7 @@ private void setFileOwner(Path path) throws IOException {
             pillar.put("_mgr_channels_items_name", "mgr_channels_new");
             pillar.put("mgr_channels_new", chanPillar);
 
-            ret.put(State.apply(Arrays.asList(ApplyStatesEventMessage.CHANNELS),
+            ret.put(State.apply(Arrays.asList(ApplyStatesEventMessage.CHANNELS, ApplyStatesEventMessage.PACKAGES),
```

Comment:
I think this will execute more than we want. I would suggest to move the state we want to execute into the channels/init.sls file.